packages/wrangler/src/images/local.ts

```diff
@@ -0,0 +1,187 @@
+import { File } from "buffer";
+import sharp from "sharp";
+import { z } from "zod";
+import type { ImageInfoResponse } from "@cloudflare/workers-types/experimental";
+import type { Sharp } from "sharp";
+
+const Transform = z.object({
+	imageIndex: z.number().optional(),
+	rotate: z.number().optional(),
+	width: z.number().optional(),
+	height: z.number().optional(),
+});
+
+const Transforms = z.array(Transform);
+
+export async function imagesLocalFetcher(request: Request): Promise<Response> {
+	const data = await request.formData();
+
+	const body = data.get("image");
+	if (!body || !(body instanceof File)) {
+		return errorResponse(400, 9523, "ERROR: Expected image in request");
+	}
+
+	const transformer = sharp(await body.arrayBuffer(), {});
+
+	const url = new URL(request.url);
+
+	if (url.pathname == "/info") {
+		return runInfo(transformer);
+	} else {
+		const badTransformsResponse = errorResponse(
+			400,
+			9523,
+			"ERROR: Expected JSON array of valid transforms in transforms field"
```

Comment:
how can we make this err message more useful for users so they know how to action? Should we maybe phrase this as `Expected ... but got .... Please make sure ...`. Also what is a valid transform? Is that documented somewhere? If so, we could add the docs link or a tl;dr so users don't have to switch context